### PR TITLE
Fixing lower end of the latest silverstripe issues

### PR DESCRIPTION
**Pull request by spekulatius**

### CODE REVIEW

**Comment by xabbuh** [comment]
You are still missing the lower bound for the `3.2.x` versions.

File: silverstripe/framework/SS-2015-026-1.yaml [hunk 1]
```diff
@@ -4,7 +4,7 @@ cve:       ~
 branches:
     3.0.x:
         time:     2015-11-11 14:31:00
-        versions: [>=3.0.0,<=3.0.14]
+        versions: [>=3.0.0,<=3.0.15]
```
Comment (xabbuh) [inline silverstripe/framework/SS-2015-026-1.yaml]: What about simply using `<3.1.0` here? It seems to me that `3.0.x` versions don't receive any fixes at all.

File: silverstripe/framework/SS-2015-027-1.yaml [hunk 2]
```diff
@@ -4,7 +4,7 @@ cve:       ~
 branches:
     3.0.x:
         time:     2015-11-13 10:30:00
-        versions: [>=3.0.0,<=3.0.14]
+        versions: [>=3.0.0,<=3.0.15]
```
Comment (xabbuh) [inline silverstripe/framework/SS-2015-027-1.yaml]: same here

**Comment by spekulatius** [comment]
Hello @xabbuh 

Isn't that considered 3.2.0 automatically?

Spek

**Comment by xabbuh** [comment]
@spekulatius No (actually that's what @stof was talking about in https://github.com/FriendsOfPHP/security-advisories/pull/118#discussion_r45588913).

**Comment by spekulatius** [comment]
ahhhh, okay @xabbuh   Now I got it. My morning coffee hasn't kicked in yet.

**Comment by xabbuh** [comment]
@spekulatius Well, I think you should nonetheless use `<3.1.0` as the upper bound for the `3.0.x` versions (the fix only mentions `3.1.x`).

**Comment by spekulatius** [comment]
okay, I can remove the whole 3.0.x section.

**Comment by spekulatius** [comment]
@xabbuh are you happy with this now?

File: silverstripe/framework/SS-2015-026-1.yaml [hunk 3]
```diff
@@ -4,11 +4,11 @@ cve:       ~
 branches:
     3.0.x:
         time:     2015-11-11 14:31:00
-        versions: [>=3.0.0,<=3.0.14]
+        versions: [>=3.0.0,<3.1.0]
     3.1.x:
         time:     2015-11-11 14:31:00
-        versions: [>3.1.0,<=3.1.16]
+        versions: [>=3.1.0,<3.2.0]
```
Comment (stof) [inline silverstripe/framework/SS-2015-026-1.yaml]: this is wrong. The silverstripe announcement says that 3.1.16 is fixing the issue too

Comment (spekulatius) [inline silverstripe/framework/SS-2015-026-1.yaml]: @stof indeed, I fixed this

**Comment by stof** [comment]
:+1:

**Comment by spekulatius** [comment]
:)

**Comment by spekulatius** [comment]
@xabbuh what do think? Are we good to merge this in?

**Comment by xabbuh** [comment]
@spekulatius Yes, but I am not a maintainer here so we have to wait for @fabpot :)

**Comment by spekulatius** [comment]
Yupp, thats fine - I just want to make sure we are all happy :)

**Pull request merged**